test/unit/specs/stores/search-store.spec.js

```diff
@@ -409,38 +409,6 @@ describe('Search Store', () => {
       }
     )
 
-    /**
-     * Changing the search type to ALL_MEDIA does not fire the watcher that clears the filters
-     * in tests, but does fire it in the app. TODO: Figure out why???
-     */
```

Comment:
Why remove this test?

Reply:
Explained here: https://github.com/WordPress/openverse-frontend/pull/1259#discussion_r849371816